.vsts-dotnet-ci.yml

```diff
@@ -0,0 +1,146 @@
+trigger:
+- master
+- exp/*
+
+phases:
+- phase: FullOnWindows
+  displayName: "Build on Windows using full MSBuild"
+  queue: 'Hosted VS2017'
+  steps:
+  - script: set
```

Comment:
Is this step needed?

Reply:
Not strictly, but it's been very useful in our official build to debug what's going on in the environment when things are really confusing. I'll remove it.

Reply:
Yeah, the experience for PR builds, where you can't publish build artifacts (yet) and capture binlogs, is pretty bad. They (VSTS) have an active issue to improve this.